src/main/features/mediation/routes/mediation-disagreement.ts

```diff
@@ -12,8 +12,21 @@ import { User } from 'main/app/idam/user'
 import { MediationDraft } from 'mediation/draft/mediationDraft'
 import { FreeMediation, FreeMediationOption } from 'main/app/forms/models/freeMediation'
 import { Claim } from 'claims/models/claim'
+import { FeatureToggles } from 'utils/featureToggles'
+import { LaunchDarklyClient } from 'shared/clients/launchDarklyClient'
+
+async function isEnhancedMediationJourneyEnabled () {
+  const featureToggles: FeatureToggles = new FeatureToggles(new LaunchDarklyClient())
+  if (await featureToggles.isEnhancedMediationJourneyEnabled()) {
```

Comment:
we can just return featureToggles.isEnhancedMediationJourneyEnabled()

Reply:
Or Just 'enhancedMediationJourney' as this is set at application level.